.github/workflows/integration_test.yml

```
@@ -43,9 +44,10 @@ jobs:
             --exclude wearable_rotary image_picker camera webview_flutter \
               video_player permission_handler geolocator battery connectivity \
               device_info package_info sensors share wifi_info_flutter \
-              google_maps_flutter tizen_app_control url_launcher network_info_plus
+              google_maps_flutter tizen_app_control url_launcher network_info_plus \
+              flutter_splash_tizen
         # The following packages are excluded from tests:
-        #   wearable_rotary, image_picker: no tests.
+        #   wearable_rotary, image_picker: no tests, flutter_splash_tizen: runs manually.
```

This package doesn't have a test.
```suggestion
        #   wearable_rotary, image_picker, flutter_splash_tizen: no tests.
```